analysis/analysis_test.go

```diff
@@ -92,33 +91,28 @@ func TestApplicationAnalysis(t *testing.T) {
 			// Prepare and submit the analyze task.
 			// tc.Task.Addon = analyzerAddon
 			tc.Task.Application = &api.Ref{ID: tc.Application.ID}
-			taskData := tc.Task.Data.(addon.Data)
+			taskData := tc.Task.Data
```

Comment:
Recommend reverting this.  Dealing with the Task.Data as a map will decouple from versions go-mod'd in from _latest_ but just a brittle (but less dangerous).  Not sure what the solution is.  For 0.4 and 0.5 (multi-provider) there are no changes to the addon-analyzer `Data` and it is not anticipated to change in future PRs.